CONTRIBUTING.md

```diff
@@ -1,36 +1,56 @@
+
+ 
+ 
+## Step-by-step Process for submission
+
+1. Set up a GitHub account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using GitHub. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
+2. Fork the [source repository](http://github.com/NESCent/popgenInfo) on GitHub using the `Fork` button. 
+3. Clone the repository to your machine using `git clone https://github.com/YourAccount/popgenInfo.git` on the command line. The URL used here is in the HTTPS clone url section on the popgenInfo GitHub page. Keep your clone synced up by following Step 3 of [this tutorial](https://help.github.com/articles/fork-a-repo/). 
+4. Then, from the command line, move to the popgenInfo directory using `cd popgenInfo`.
 5. Create a branch for your changes using `git checkout -b Your_branch_name`.
-6. Move your workflow vignette in `.Rmd` format into the **use/** directory.
+6. Move your  vignette in `.Rmd` format into the **use/** directory.
 8. Add the file to git using `git add use/yourfilename.Rmd`.
 9. Then commit this using `git commit` and add some short text describing what your file is.
 10. Then run `git push origin Your_branch_name`.
-11. To do a pull request, navigate to your branch on github. Click `Compare & review` or the pull request icon.
+11. To do a pull request, navigate to your branch on GitHub. Click `Compare & review` or the pull request icon.
 12. Once you have reviewed the changes, click `Create pull request`. Once the request is approved, the file will be automatically sent to a remote server that compiles the document into a html file.
-13. Then open the [`WORKFLOWS.md`](https://github.com/nescent/popgenInfo/tree/master/WORKFLOWS.md) file using the edit button if on the github website (or alter it your branch of the repository that has already been cloned on your own machine) and add the name of your tutorial along with a link to the tutorial e.g. the link will be `Tutorial_name.html`. Finally click `Propose file change` and then `Create pull request` to request that the file changes be included if you are altering it on github.
 
-## How do I submit datasets?
+## Guidelines for approval
+
+ - Vignette must be able to successfully run examples with available packages and data (this is automatically checked)
+ - At least two people other than the vignette writer must approve the pull request
+
+## Best practices
+
+ - Use the [template](https://github.com/nescent/popgenInfo/tree/master/TEMPLATE.Rmd) to get started
+ - Give your vignette a unique name by using the date of your first commit (e.g. `2015-06-16_Parentage_Microsatellite.Rmd`) would be a vignette that was started on June 16th, 2015) 
+ - List authors at the bottom of the vignette
+ - Use small examples and encourage readers to explore with bigger examples
+ - Submit your vignette early to get feedback
```

Comment:
Add a statement to indicate on the pull request description if the submission isn't considered ready for merging but primarily there for obtaining feedback?